### ISSUE TITLE
Improve application accessibility

### ISSUE THREAD

**Issue by MatthewKosloski** [MatthewKosloski/polysent#6]
The web application lacks accessibility features that make the website a11y-friendly, such as navigating through dropdown menus with the keyboard.
